src/liger_kernel/transformers/swiglu.py

```diff
@@ -38,3 +38,27 @@ def __init__(self, config):
     def forward(self, x):
 
         return self.w2(LigerSiLUMulFunction.apply(self.w1(x), self.w3(x)))
+
+
+class LigerPhi3SwiGLUMLP(nn.Module):
```

Comment:
So Llama has its own MLP implementation here, but its named very generally. I went for a model-specific name here, but open to suggestions.

Reply:
yeah. this is necessary